src/lib/docs/Utilities/SigningRequest.md

```diff
@@ -9,20 +9,28 @@ order: 5
 
 # Signing Request
 
-Signing Requests are a data type used in the Antelope ecosystem to facilitate the seamless transfer of transaction details in a compressed string format between applications and signature providers. Designed to encapsulate and transport transaction data, they can be shared through various methods, such as QR codes and hyperlinks.
+Signing Requests are a data type used in the Antelope ecosystem to facilitate the transferring of transaction related data between applications for the purposes of generating a signature. This is done using a compressed string format designed to encapsulate and make it easier to transport data, which can be communicated through various methods such as QR codes and hyperlinks.
 
-Two types of signing requests exist:
+The Wharfkit Signing Request library provides a convenient way to interact with this type of data through the `@wharfkit/signing-request` package. The `SigningRequest` class exported by this package offers methods to easily encode or decode transactions, and resolve incomplete transaction data.
 
-- **Transaction Signing Requests** - These requests are used to request a signature for a specific transaction or action meant to be broadcasted on-chain.
-- **Identity Signing Requests** - These requests are used to request an identity proof from a wallet. This request is fulfilled by the wallet by signing an off-chain transaction that contains the identity information.
+## Creation
 
-The Wharfkit Signing Request library provides a convenient way to interact with these signing requests through the `@wharfkit/signing-request` package. The `SigningRequest` wrapper which is exported by that package offers methods to easily create, parse and resolve signing requests.
+Signing requests are used primarily for two purposes:
```

Comment:
I feel like this part belongs in the intro and not in the `Creation` section.

Reply:
Maybe it just needs to be worded to be "Signing requests are created for two purposes".